### ISSUE TITLE
Run QEMU tests in parallel

### ISSUE THREAD

**Issue by gsingh93** [pwndbg/pwndbg#1741]
The QEMU tests are very slow, and we have to keep bumping the timeout. We should parallelize them just like we've done for the normal tests. We may need to implement a workaround for this first: https://github.com/pwndbg/pwndbg/issues/1587

Also, maybe this should be done along with or after #1687

**Comment by intrigus-lgtm** [comment]
Hasn't this been fixed?

**Comment by gsingh93** [comment]
I don't think so, the QEMU testing scripts are different than the other GDB tests and I believe they still run serially.

**Comment by intrigus-lgtm** [comment]
Ah, you're probably right.
I somehow thought that this commit also made the tests parallel, but that's not true:
https://github.com/pwndbg/pwndbg/commit/ee0377d3bf6d665b104d4de9a2762dc1b152449d
